db/migrate/20200904005346_create_genres.rb

```diff
@@ -0,0 +1,13 @@
+class CreateGenres < ActiveRecord::Migration[5.2]
+  def change
+    create_table :genres do |t|
+      t.string :name
+      t.timestamps
+    end
+
+    create_table :books_genres, id: false do |t|
+      t.belongs_to :book
+      t.belongs_to :genre
```

Comment:
Likewise, we can add null protections here

```ruby
t.belongs_to :book, null: false
t.belongs_to :genre, null: false
```

where possible we should always add null protections, to ensure we aren't allowing trash data to get into our database